CONTRIBUTING.md

```diff
@@ -143,6 +143,13 @@ package on the public dev feed, e.g. [@azure/mcp@0.0.10-beta.4799791](https://de
 
 Only manual runs of the pipeline sign and publish packages.  Building `main` or `hotfix/*` will publish to `npmjs.com`, all other refs will publish to the [public dev feed](https://dev.azure.com/azure-sdk/public/_artifacts/feed/azure-sdk-for-js).
 
+Packages published to the npmjs.com will always use the `@latest` [dist-tag](https://docs.npmjs.com/downloading-and-installing-packages-locally#installing-a-package-with-dist-tags).
+
+Packages published to the dev feed will use:
+- `@latest` for the latest official/release build
+- `@dev` for the latest CI build of main
+- `@pre` for any arbitrary pipeline run or feature branch build
```

Comment:
Do we need a tag for this? I would suggest just continuing with dev tag or no tag if people are publishing arbitrary versions.

Reply:
there's no concept of `no tag` in npm releases.

> Publishing a package sets the latest tag to the published version unless the --tag option is used.

Reply:
To differentiate official releases from unofficial and latest main from everything else, we need at least 3 tags

We could go with 

official -> `@stable`
main -> `@latest`
other -> `@dev`

Reply:
But this would differ from the official feed where @latest is always stable and the only tag we publish

Reply:
What does latest CI build of main mean? Does this include PRs that are not merged?

Reply:
It would be nice if dev (or unreleased) meant builds from commits to main, not CI pipeline runs. I think CI pipeline should be a lower-designation than dev/unreleased

Reply:
@hallipr partner teams would love to have this merged ASAP.  Can we get this part resolved today

Reply:
CI as in the original VSO definition.  The real build of main because a change was pushed.

Reply:
https://learn.microsoft.com/en-us/devops/develop/what-is-continuous-integration

Reply:
thanks @hallipr.  Could you just tweak the text to convey it's the real build of main (and not a CI build from a random PR from the internet)